packages/core/data-transfer/lib/providers/local-file-destination-provider.ts

```diff
@@ -131,147 +134,133 @@ class LocalFileDestinationProvider implements IDestinationProvider {
   }
 
   #getMetadataStream() {
-    const metadataPath = path.join(this.options.file.path, 'metadata.json');
-
-    // Transform streams
-    const transforms: Writable[] = this.#getDataTransformers({ jsonl: false });
-
-    // FS write stream
-    const fileStream = fs.createWriteStream(metadataPath);
-
-    // Full pipeline
-    const streams = transforms.concat(fileStream);
-
-    return chain(streams);
+    return createTarEntryStream(this.#archive!, () => 'metadata.json');
   }
 
   getSchemasStream() {
-    const filePathFactory = createFilePathFactory(this.options.file.path, 'schemas');
-
-    // Transform streams
-    const transforms: Writable[] = this.#getDataTransformers();
+    // const filePathFactory = createFilePathFactory(this.options.file.path, 'schemas');
```

Comment:
Are these comments left for future use or safe to delete?

Reply:
I forgot to remove them, my bad